packages/preview/bellbird-udesc-paper/0.0.1/README.md

```diff
@@ -0,0 +1,157 @@
+# paper-udesc
+
+This is an academic template for dissertations and theses, based on the formatting guidelines provided by the **Universidade do Estado de Santa Catarina (UDESC)**. This is **not an official project**.
+
+All **pre-textual elements** have been automated, along with a few style conventions. The goal of this template is to simplify the formatting process for academic writing according to institutional norms, allowing you to focus on content rather than layout. If you'd like to help, feel free to send me an email at: lucas [at] bublitz.art.br.
+
+## Warings
+
+- Typst currently does **not** allow modification of template code hosted under `@preview`.
+- If you need to edit internal textual elements generated by the `paper-udesc` function, you must include the `.typ` file **locally** in your project.
+  - The source file is available at: [https://github.com/lucas-bublitz/bellbird-udesc-paper](https://github.com/lucas-bublitz/bellbird-udesc-paper)
+- **Arial** and **Times New Roman** fonts are **not bundled** with the Typst compiler by default — not even in the web version.
+  - Therefore, they must be added manually.
+  - Due to package guidelines, it's **not allowed** to include external font files directly in a template.
+- In the Typst web app, local font files are automatically recognized when present in the project.
+  - However, the font must also be **installed on your operating system** to render correctly.
+    - On Windows, these fonts are typically pre-installed.
```

Comment:
This is only true when running Typst outside of the web app, I think it needs to be stated more explicitly. Also, it is possible to not install the fonts globally and to use `--font-path`, it may be worth mentioning.